docs/api/features/rankd.rst

```diff
@@ -43,42 +43,40 @@ A one dimensional ranking of features utilizes a ranking algorithm that takes in
 
 .. image:: images/rank1d_shapiro.png
 
-
 Rank 2D
 -------
 
-A two dimensional ranking of features utilizes a ranking algorithm that takes into account pairs of features at a time (e.g. joint plot analysis). The pairs of features are then ranked by score and visualized using the lower left triangle of a feature co-occurence matrix.
+A two-dimensional ranking of features utilizes a ranking algorithm that takes into account pairs of features at a time (e.g. joint plot analysis). The pairs of features are then ranked by score and visualized using the lower left triangle of a feature co-occurence matrix.
 
-The default ranking algorithm is covariance, which attempts to compute the mean value of the product of deviations of variates from their respective means. Covariance loosely attempts to detect a colinear relationship between features.
+By default, the ``Rank2D`` visualizer utilizes the Pearson correlation score to detect colinear relationships. 
```

Comment:
Since I moved the text, code, and image for the Pearson example above the covariance example, here it looks like I deleted the text describing `Rank2D` when used with `covariance` as the ranking algorithm. However, I simply moved it below, which you'll see further down.